nemo/collections/asr/models/classification_models.py

```diff
@@ -690,14 +690,15 @@ def training_step(self, batch, batch_idx):
         train_mse = self._mse(preds=logits, target=targets)
         train_mae = self._mae(preds=logits, target=targets)
 
-        tensorboard_logs = {
-            'train_loss': loss,
-            'train_mse': train_mse,
-            'train_mae': train_mae,
-            'learning_rate': self._optimizer.param_groups[0]['lr'],
-        }
+        self.log_dict(
+            {
```

Comment:
You'll lose multi data loader support for classification models with this (GSC has val and test sets)

Reply:
Thanks @titu1994 I might be missing something here, all I've done is assignment of the variable and combined it into the `self.log_dict` call (primarily because the idea of `tensorboard_logs` has been deprecated for a while in Lightning).